lib/melissa_data/web_smart/response_processor.rb

```diff
@@ -1,39 +1,40 @@
 module MelissaData
   module WebSmart
     module ResponseProcessor
-      def process_property(response)
-        codes = codes(response)
+      require 'yaml'
+      CODE_TYPES = [:property, :address]
```

Comment:
In order to add another type of response processing to get these errors in the future the steps are quite simple. First, you would add the symbol for the request name here